docs/source/docker/examples/academic_datasets.rst

```diff
@@ -51,18 +51,28 @@ selects a diverse set of frames:
 
 .. code-block:: python
 
+    # in this example we use a diversifying selection strategy (CORESET)
+
     client.schedule_compute_worker_run(
         worker_config={
             "enable_corruptness_check": False,
             "remove_exact_duplicates": True,
             "enable_training": False,
             "pretagging": False,
-            "pretagging_debug": False,
-            "method": "coreset",
-            "stopping_condition": {
-                "n_samples": 500000,
-                "min_distance": -1
-            }
+            "pretagging_debug": False
+        },
+        selection_config = {
+            "nSamples": 500000,
```

Comment:
feels weird that its `nSamples` and not stopping condition as its with min_distance.

Reply:
We cannot really have a "stopping condition" struct anymore, as the minimum distance stopping condition was moved to the DIVERSIFY strategy, as it only applies to that strategy and is not global anymore.